src/convertToObject.js

```diff
@@ -1,12 +1,40 @@
 'use strict';
 
+/**
+ * @param {string} sourceString
+ *
+ * @return {object}
+ */
+'use strict';
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const parts = sourceString.split(';');
+
+  parts.forEach((str) => {
+    const cleanStr = str.trim();
+    const colonIndex = cleanStr.indexOf(':');
+
+    if (colonIndex === -1) {
+      return;
+    }
+
+    const property = cleanStr.slice(0, colonIndex).trim();
+    const value = cleanStr.slice(colonIndex + 1).trim();
+
+    if (property && value) {
+      obj[property] = value;
+    }
+  });
+
+  return obj;
 }
 
 module.exports = convertToObject;
+
+module.exports = convertToObject;
```

Comment:
The `module.exports = convertToObject;` statement is duplicated. Remove one to ensure that `convertToObject` is only exported once.

Reply:
Remove duplicates 
<img width="551" alt="image" src="https://github.com/user-attachments/assets/840b66a5-a7fd-4daa-ad93-f5ac8a9c183d">